### ISSUE TITLE
FC3 module not exporting because of NS430 control GLOW

### ISSUE THREAD

**Issue by jdahlblom** [DCS-Skunkworks/dcs-bios#503]
```
2023-12-05 08:58:09.698 ERROR   Lua::Config (Main): Call error LuaExportAfterNextFrame:[string "C:\Users\username\Saved Games\DCS.openbeta\Scripts\DCS-BIOS\lib\modules\Module.lua"]:340: attempt to index local 'dev0' (a nil value)
stack traceback:
	[C]: ?
	[string "C:\Users\username\Saved Games\DCS.openbeta\Scripts\DCS-BIOS\lib\modules\Module.lua"]:340: in function 'hook'
	[string "C:\Users\username\Saved Games\DCS.openbeta\Scripts\DCS-BIOS\lib\BIOSStateMachine.lua"]:70: in function 'queue_module_data'
	[string "C:\Users\username\Saved Games\DCS.openbeta\Scripts\DCS-BIOS\lib\BIOSStateMachine.lua"]:151: in function 'step'
	[string "C:\Users\username\Saved Games\DCS.openbeta\Scripts\DCS-BIOS\BIOS.lua"]:226: in function 'LuaExportAfterNextFrame'
```

This is caused by the ```dcs-bios``` ```GLOW``` control in NS430 :
```
NS430:defineIndicatorLight("GLOW", 70, "NS430", "Glow (green)")
```

Mod ```NS430``` and ```mainpanel_init.lua``` lists a gauge glow as follows :
```
local controllers = LoRegisterPanelControls()
	
glow                    = CreateGauge()
glow.arg_number         = 70
glow.input              = {0,1}
glow.output             = {0,1}
glow.controller         = controllers.glow
```
The NS430 has what looks like a LED in the left top side of the panel. I haven't been able to get this LED to light up. I can't find information in the NS430 pdf about it either. Is is lit when plane is on course or when communicating with landing approach, I have no idea.

When ending the mission the dcs-bios values are updated.

https://github.com/DCS-Skunkworks/dcs-bios/pull/499

**Comment by charliefoxtwo** [comment]
Was this fixed in #499? Can we close this?

**Comment by jdahlblom** [comment]
Yes it was fixed but GLOW control is missing. So NS430 & FC3 works but that mysterious LED which must the the GLOW control is not implemented.
